lesson_4/src/main/java/org/example/models/Entity.java

```diff
@@ -0,0 +1,13 @@
+package org.example.models;
+
+public abstract class Entity {
+    protected int Id;
```

Comment:
It is better to follow naming conventions for field names, and use id instead of Id